.github/workflows/firmware.yml

```diff
@@ -40,8 +54,8 @@ jobs:
       - name: build target ${{ matrix.target }}
         id: compile
         run: |
-          git checkout -b patched && git checkout ${{ steps.extract_branch.outputs.branch }}
-          make BROKEN=1 GLUON_TARGETS=${{ matrix.target }} GLUON_RELEASE=${{ steps.vars.outputs.tag }} V=s
```

Comment:
Couldn't this cause problems when a build starts on 17:59 and the Release get's different names?

Reply:
I agree, it does. The naming scheme is something that I want to rework in a follow-up PR anyway, but for now I don't want to modify the existing one.

Reply:
I think it should be generated centrally and passed on as a variable.

Reply:
The "release" name here is for (temporary) PR images only. Even if this is globally defined and two PRs start at the same time, you will have identical temporary image "versions" there as well.

The only way to solve this for good is to include some kind of unique identifer (e.g. a git commit hash).

Reply:
But where does the release name for regular releases come from?

Reply:
The release name for regular releases comes from the tag name:
https://github.com/freifunkMUC/site-ffm/blob/c401dc063e9a40d56823b26f0f3f6eebda5eec17/Makefile#L35-L37

It literally just copies the name of the tag and uses it as release version like it did before.

Reply:
Here is a sample for a tag-triggered Actions run: https://github.com/grische/site-ffm/actions/runs/1087511249

Reply:
Okay, then it works differently than I expected.

Reply:
I initially started with a rather complex Actions step to make sure that the `GLUON_RELEASE` is consistent until @goligo mentioned that it duplicates the logic of the Makefile. Saved a lot of headache.